src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/WorkloadsDetectedOSTypeModel.java

```diff
@@ -0,0 +1,102 @@
+package org.jboss.xavier.analytics.pojo.output.workload.summary;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.*;
+
+@SqlResultSetMapping(
+        name = "mappingWorkloadsDetectedOSTypeModels",
+        classes = @ConstructorResult(
+                targetClass = WorkloadsDetectedOSTypeModel.class,
+                columns = {
+                        @ColumnResult(name = "osName", type = String.class),
+                        @ColumnResult(name = "total", type = Integer.class)
+                }
+        )
+)
+
+@NamedNativeQuery(
+        name = "WorkloadsDetectedOSTypeModel.calculateWorkloadsDetectedOSTypeModels",
+        query = "select coalesce(wm.os_name, 'unknown') as osName, count(*) as total \n" +
```

Comment:
In the `workload_model.vms` column there's the number of times the workload has been detected and it should drive the donut chart as well.
```suggestion
        query = "select coalesce(wm.os_name, 'unknown') as osName, sum(wm.vms) as total \n" +
```

Reply:
Thanks @mrizzi. I've just added your suggested changes.